### PR TITLE
implement basic http server, add Dockerfile, docker-compose.yml

### PR DESCRIPTION
**Pull request by omrishtam**
Signed-off-by: Omri Shtamberger <omrishtam@gmail.com>
Dockerfile, docker-compose.yml are generated with the Docker extension for VSCode

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,5 +1,22 @@
 package main
 
-func main() {
+import (
+	"net/http"
+	"log"
+)
+
+const (
+	address string = ":8080"
```
Comment (omerzamir) [inline main.go]: why arent we get it from env variable?

**Review by omerzamir — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (omrishtam) [reply main.go]: 🤷‍♂️

Comment (omrishtam) [reply main.go]: And now?

**Review by omerzamir — APPROVED** [review]
should address end with ':' or should we concatenate it?

**Comment by omrishtam** [comment]
> should address end with ':' or should we concatenate it?

it's a shortcut for 0.0.0.0:port

**Review by omerzamir — APPROVED** [review]
LGTM

**Pull request merged**